addon/components/o-s-s/attribute-text.stories.js

```diff
@@ -0,0 +1,73 @@
+import hbs from 'htmlbars-inline-precompile';
+
+export default {
+  title: 'Components/OSS::Attribute::Text',
+  component: 'text',
+  argTypes: {
+    label: {
+      type: { required: true },
+      description: 'The value of the label',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'text' }
+    },
+    value: {
+      type: { required: false },
```

Comment:
The default value of the `required` is false, I think you can remove it when it `false`